src/Runner.Worker/OSWarningChecker.cs

```diff
@@ -83,14 +96,15 @@ public async Task CheckOSAsync(IExecutionContext context)
             {
                 Trace.Error("An error occurred while testing .NET 8 compatibility'");
                 Trace.Error(ex);
-                context.Global.JobTelemetry.Add(new JobTelemetry() { Type = JobTelemetryType.General, Message = $".NET 8 OS compatibility test encountered exception type '{ex.GetType().FullName}', message: '{ex.Message}', process output: '{GetShortOutput(output)}'" });
+                context.Global.JobTelemetry.Add(new JobTelemetry() { Type = JobTelemetryType.General, Message = $".NET 8 OS compatibility test encountered exception type '{ex.GetType().FullName}', message: '{ex.Message}', process output: '{GetShortOutput(context, output)}'" });
             }
         }
 
-        private static string GetShortOutput(List<string> output)
+        private static string GetShortOutput(IExecutionContext context, List<string> output)
         {
+            var length = context.Global.Variables.System_DotNet8CompatibilityOutputLength ?? 200;
```

Comment:
given we knows that 200 is not enough, should we bump it higher by default? :D

Reply:
might be enough after publishing non-trimmed